simcon_project/simcon_project/urls.py

```diff
@@ -15,7 +15,10 @@
 """
 from django.contrib import admin
 from django.urls import path
-from users.views import RedirectFromLogin, StudentView, ResearcherView, ViewResponse, UpdateOverallResponseFeedback, UpdateNodeResponseFeedback, ResearcherSettingsView, ResearcherUserView, StudentSettingsView,StudentUserView
+from users.views import RedirectFromLogin, StudentView, ResearcherView, ViewResponse,\
```

Comment:
Lol peeps are going back and forth on this. I personally like this style best though

Reply:
I like multi-line best too